modules/nw-sriov-configure-exclude-topology-manager.adoc

```diff
@@ -155,7 +163,7 @@ test-deployment-sriov-76cbbf4756-k9v72   1/1     Running   0          45h
 
 . Open a debug session with the target pod to verify that the SR-IOV network resources are deployed to a different node than the memory and CPU resources.
 
-.. Open a debug session with the pod by running the follow command, replacing <pod_name> with the target pod name. 
+.. Open a debug session with the pod by running the follow command, replacing <pod_name> with the target pod name.
```

Comment:
I only saw this due to the whitespace change, but there's a typo: "follow" should be "following".

Reply:
Hi @kowen-rh . Are you performing a peer review for this PR? If so, would you be OK to add a label?

"/remove peer-review-in-progress"

Reply:
Not at the moment, no -- just happened to see that when I glanced over it. :)

Reply:
Thanks, will fix that typo!